legal-api/src/legal_api/services/namex.py

```diff
@@ -64,6 +68,46 @@ def query_nr_number(identifier: str):
 
         return nr_response
 
+    @staticmethod
+    def update_nr(nr_json):
```

Comment:
Does this method have to be declared as async?

Reply:
I don't believe so, I ran some tests against this for various filings and it seems to work fine. I also haven't really used async for python so I am not familiar what requirements in this case should warrant its usage.

Reply:
Per discussion with Thor, this method simply blocks on its POST call (and runs in its own thread), so it doesn't need to be async (and it is not awaited upon anyway)... In short, this code is fine. I'm resolving this conversation.